test/functional/mvn-plugin.test.js

```diff
@@ -1,4 +1,4 @@
-var test = require('tap-only');
+var test = require('tap').test;
```

Comment:
I love tap-only - would prefer to leave it. It's useful when debugging - can easily make a single test to run without commenting out code

Reply:
ok let me bring it back, one sec

Reply:
🙏